src/www/status_graph.php

```diff
@@ -48,6 +50,24 @@
     }
 }
 
+//Create array of hostnames from DHCP
+foreach ($interfaces as $ifname => $ifarr) {
+	if (isset($config['dhcpd'][$ifname]['staticmap'])) {
+		foreach($config['dhcpd'][$ifname]['staticmap'] as $entry) {
+			if (htmlentities($entry['hostname']) !== "") {
```

Comment:
probably better to check !empty($entry['hostname']) and leave the entities in the line below.